src/NavDropdown.js

```diff
@@ -57,28 +58,36 @@ const NavDropdown = React.forwardRef(
       ...props
     },
     ref,
-  ) => (
-    <Dropdown ref={ref} {...props} as={NavItem}>
-      <Dropdown.Toggle
-        id={id}
-        eventKey={null}
-        active={active}
-        disabled={disabled}
-        childBsPrefix={bsPrefix}
-        as={NavLink}
-      >
-        {title}
-      </Dropdown.Toggle>
+  ) => {
+    const onSelectCtx = useContext(SelectableContext);
+
+    const handleSelect = useEventCallback((key, event) => {
+      if (onSelectCtx) onSelectCtx(key, event);
+    });
+
+    return (
+      <Dropdown ref={ref} onSelect={handleSelect} {...props} as={NavItem}>
```

Comment:
i wonder if we want to make `as` customizable in this context, then.

with this change, rendering a `<Dropdown>` with whatever styling inside a `<Nav>` now won't do the same thing as rendering a `<NavDropdown>`

which is probably okay, but maybe a bit more customizability here would help